port/scorecard/schema.go

```diff
@@ -70,6 +80,13 @@ func ScorecardSchema() map[string]schema.Attribute {
 			MarkdownDescription: "The title of the scorecard",
 			Required:            true,
 		},
+		"levels": schema.ListNestedAttribute{
+			MarkdownDescription: "The Levels of the scorecard",
+			Optional:            true,
```

Comment:
I think we can be more descriptive here. 
Also I don't see any additional docs and example on how to setup the custom scorecards.

you can add it to the markdown below, and run `make gen-docs` to see it generates